spec/manipulate_file_spec.rb

```diff
@@ -1,18 +1,31 @@
 require 'manipulate_file'
 
-describe '#get_first_line' do
-  it 'read the first line of the file' do   
-    @content = "Test 1,2,3"
+describe ManipulateFile do 
+  describe '#get_first_line' do
+    it 'read the first line of the file' do   
+      @content = "Test 1,2,3"
 
-    main = ManipulateFile.new("test.txt")
+      main = ManipulateFile.new("test.txt")
 
-    expect(main.get_first_line).to eq(@content)
-  end
+      expect(main.get_first_line).to eq(@content)
+    end
+      
+    it 'treat error when file doesn`t exist' do
+      arq = "teste"
+      
+      expect { ManipulateFile.new(arq) }.to raise_error("File doesn't exist.")
+
+    end
+  end 
+
+  describe '#mount_json' do
+    it 'create a json object' do   
+      
+      main = ManipulateFile.new("test.txt")
     
-  it 'treat error when file doesn`t exist' do
-    arq = "teste"
-     
-    expect { ManipulateFile.new(arq) }.to raise_error("File doesn't exist.")
+      obj = { main.name => { "lines" => main.file.length} }
```

Comment:
@iraline não está errado a forma que você fez aqui, porém é mais comum que o esperado dos testes sejam mais explicitos, isso também evita falsos positivos

No caso aqui, `main.name` deveria ser substituido pelo nome do seu arquivo e o `main.file.length` pelo valor retornado

Exemplo: 
```suggestion
      obj = { "nome do arquivo" => { "lines" => 33} }
```